tools/azure-sdk-tools/devtools_testutils/__init__.py

```diff
@@ -60,6 +61,7 @@
     "RandomNameResourceGroupPreparer",
     "CachedResourceGroupPreparer",
     "PowerShellPreparer",
```

Comment:
Remove?

Reply:
i'd say keep this till next MQ for backward compat (i'm assuming hence the alias)